CHANGES.md

```diff
@@ -1,6 +1,8 @@
 Unreleased
 ----------
 
+- Fixed dependency on installed packages (#4170, @bobot)
```

Comment:
This could be a bit more clear. E.g. "Allow (package <pkg>) whenever <pkg> is an installed package".